api/scrpts/fix_addresses.py

```diff
@@ -0,0 +1,38 @@
+import os
+import psycopg2
+from psycopg2 import Error
+from notd.chain_utils import normalize_address
+
+try:
+    # Connect to an existing database
+    connection = psycopg2.connect(user=os.environ['DB_USERNAME'],
+                                  password=os.environ['DB_PASSWORD'],
+                                  host=os.environ['DB_HOST'],
+                                  port=os.environ['DB_PORT'],
+                                  database=os.environ['DB_NAME'])
+
+    cursor = connection.cursor()
```

Comment:
its better to use context maangers instead of opening and closign the conenction manually. see how this is done in the project

Reply:
So this is the one i am not clear on? do you mean using await database.connection()?